client/src/Movies/MovieList.js

```diff
@@ -35,6 +36,7 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
+    <Link to={`/movies/${movie.id}`}>
```

Comment:
great job here, you could have also pulled the id from movie in the destructring above on line 36 and referenced it as just id here, either way works